server/protocols/jmap-rfc-8621/doc/specs/spec/mail/rights.mdown

```diff
@@ -64,7 +64,7 @@ Namely:
  - `Mailbox/get`, `Mailbox/query`, `Mailbox/queryChanges`, `Mailbox/changes` need to accept or return delegated.
```

Comment:
Documentation update in `server/protocols/jmap-rfc-8621/doc/specs/spec/mail/searchsnippet.mdown` too?